docs/readme-zephyr.md

```diff
@@ -163,51 +163,53 @@ example) can be used to sign images.
 
 ## Serial recovery
 
-### interface selection
+### Interface selection
 
-Serial recovery protocol can be availeble over either the serial port or the USB CDC ACM virtual serial port.
+A serial recovery protocol is available over either a serial port or an USB CDC ACM virtual serial port.
 The SMP server implementation can be enabled by ``CONFIG_MCUBOOT_SERIAL=y``.
-Kind of interface used can be set using ``BOOT_SERIAL_DEVICE`` Kconfig chose: ``CONFIG_BOOT_SERIAL_UART`` or ``CONFIG_BOOT_SERIAL_CDC_ACM`` can be selected.
-Which interface belongs to the protocol shall be set by the devicetree-chosen-node:
+To set a type of an interface, use ``BOOT_SERIAL_DEVICE`` Kconfig option and choose either ``CONFIG_BOOT_SERIAL_UART`` or ``CONFIG_BOOT_SERIAL_CDC_ACM``.
```

Comment:
``BOOT_SERIAL_DEVICE`` is Kconfig choice.
sentence should stand:
>To set a type of an interface, use ``BOOT_SERIAL_DEVICE`` Kconfig choice, select either ``CONFIG_BOOT_SERIAL_UART`` or ``CONFIG_BOOT_SERIAL_CDC_ACM``.

Reply:
```suggestion
To set a type of an interface, use ``BOOT_SERIAL_DEVICE`` Kconfig choice, and select either ``CONFIG_BOOT_SERIAL_UART`` or ``CONFIG_BOOT_SERIAL_CDC_ACM`` value.
```